baby-gru/src/utils/MoorhenMolecule.ts

```diff
@@ -226,11 +275,23 @@ export class MoorhenMolecule implements moorhen.Molecule {
         }
     }
 
+    /**
+     * Draw symmetry mates for the current molecule
+     * @param {boolean} [fetchSymMatrix=true] - Indicates whether a new symmetry matrix must be fetched from libcoot api
+     */
+    drawBiomolecule(fetchSymMatrix: boolean = true): void {
+        if (fetchSymMatrix) {
+            this.fetchBiomoleculeMatrix()
+        }
+        this.representations.forEach(representation => representation.drawSymmetry())
+    }
+
     /**
      * Draw symmetry mates for the current molecule
      * @param {boolean} [fetchSymMatrix=true] - Indicates whether a new symmetry matrix must be fetched from libcoot api
      */
     async drawSymmetry(fetchSymMatrix: boolean = true): Promise<void> {
+        if(this.biomolOn) return;
```

Comment:
Can we print a warning? Something like `console.warn('Cannot draw cryst. symm. when bio. symm. is enabled)` will suffice.

Reply:
Done in 98e02208e3be2dfe1892e1d9bf9ac019e067b9d4.